playbooks/roles/edxapp/defaults/main.yml

```diff
@@ -258,6 +258,7 @@ EDXAPP_FEATURES:
   ENABLE_INSTRUCTOR_ANALYTICS: false
   PREVIEW_LMS_BASE: "{{ EDXAPP_PREVIEW_LMS_BASE }}"
   ENABLE_GRADE_DOWNLOADS: true
+  ENABLE_COURSE_FILENAME_CCX_SUFFIX: true
```

Comment:
Note for version upgrade:

This can be set via secure vars ([example](https://github.com/edx-olive/olive-secure/search?q=ENABLE_COURSE_FILENAME_CCX_SUFFIX)), so it should be fine to drop this change when rebasing Ginkgo customizations on release branch for, e.g., Koa.

Upstream [master](https://github.com/edx/configuration/blob/master/playbooks/roles/edxapp/defaults/main.yml) doesn't reference this setting.